src/controllers/post.controller.ts

```diff
@@ -16,6 +16,16 @@ class PostController {
       next(error);
     }
   };
+
+  public getPosts = async (req: RequestWithUser, res: Response, next: NextFunction) => {
+    try {
+      const posts : BasePostDto[] = await this.postService.getPostsFromDB();
```

Comment:
Instead of calling it posts, lets keep everything consistent and call this data. res.json{ data }

Reply:
in @DLiSWE's endpoint to create posts it's called post though?

Reply:
Changed